rubicon_ml/client/__init__.py

```diff
@@ -1,7 +1,12 @@
 from rubicon_ml.client.base import Base  # noqa F401
 from rubicon_ml.client.config import Config
 
-from rubicon_ml.client.mixin import ArtifactMixin, DataframeMixin, TagMixin  # noqa F401
+from rubicon_ml.client.mixin import (
+    ArtifactMixin,
+    DataframeMixin,
+    TagMixin,
+    CommentMixin,
+)  # noqa F401
```

Comment:
I think if you move the `# noqa F401` to line 4 after `import (` the linter will pick it up and ignore these lines without needing to update the global ignore list